frontend/app/module/polling_stations/page/PollingStationListPage.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { screen } from "@testing-library/react";
+import { describe, expect, test } from "vitest";
+
+import { PollingStationListPage } from "app/module/polling_stations";
+import { render } from "app/test/unit";
+
+import { PollingStationListProvider } from "@kiesraad/api";
+
+describe("PollingStationListPage", () => {
+  test("Show polling stations", async () => {
+    render(
+      <PollingStationListProvider electionId={1}>
```

Comment:
It doesn't matter which `electionId` you provide here, since the mock will always return data.

Should we have the mock return a 404 for `electionId` `404`? Or solve this in a different way? Because then we can also build an RTL test for the "Error no polling stations data" message.

Reply:
Discussed. Test for empty list has been added. Test for 404 not because handled in different component.